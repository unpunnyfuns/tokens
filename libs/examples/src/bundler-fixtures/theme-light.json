{
  "color": {
    "primary": {
      "$type": "color",
      "$value": {
        "colorSpace": "srgb",
        "components": [
          0,
          0.498,
          1
        ],
        "alpha": 1
      }
    },
    "secondary": {
      "$type": "color",
      "$value": {
        "colorSpace": "srgb",
        "components": [
          0.498,
          0,
          1
        ],
        "alpha": 1
      }
    },
    "neutral": {
      "100": {
        "$type": "color",
        "$value": {
          "colorSpace": "srgb",
          "components": [
            0.961,
            0.961,
            0.961
          ],
          "alpha": 1
        }
      },
      "900": {
        "$type": "color",
        "$value": {
          "colorSpace": "srgb",
          "components": [
            0.129,
            0.129,
            0.129
          ],
          "alpha": 1
        }
      }
    },
    "background": {
      "primary": {
        "$type": "color",
        "$value": {
          "colorSpace": "srgb",
          "components": [
            0.961,
            0.961,
            0.961
          ],
          "alpha": 1
        }
      },
      "secondary": {
        "$type": "color",
        "$value": {
          "colorSpace": "srgb",
          "components": [
            0.129,
            0.129,
            0.129
          ],
          "alpha": 1
        }
      }
    },
    "text": {
      "primary": {
        "$type": "color",
        "$value": {
          "colorSpace": "srgb",
          "components": [
            0.129,
            0.129,
            0.129
          ],
          "alpha": 1
        }
      },
      "secondary": {
        "$type": "color",
        "$value": {
          "colorSpace": "srgb",
          "components": [
            0.961,
            0.961,
            0.961
          ],
          "alpha": 1
        }
      },
      "link": {
        "$type": "color",
        "$value": {
          "colorSpace": "srgb",
          "components": [
            0,
            0.498,
            1
          ],
          "alpha": 1
        }
      }
    },
    "button": {
      "primary": {
        "background": {
          "$type": "color",
          "$value": {
            "colorSpace": "srgb",
            "components": [
              0,
              0.498,
              1
            ],
            "alpha": 1
          }
        },
        "text": {
          "$type": "color",
          "$value": {
            "colorSpace": "srgb",
            "components": [
              0.961,
              0.961,
              0.961
            ],
            "alpha": 1
          }
        }
      },
      "secondary": {
        "background": {
          "$type": "color",
          "$value": {
            "colorSpace": "srgb",
            "components": [
              0.498,
              0,
              1
            ],
            "alpha": 1
          }
        },
        "text": {
          "$type": "color",
          "$value": {
            "colorSpace": "srgb",
            "components": [
              0.961,
              0.961,
              0.961
            ],
            "alpha": 1
          }
        }
      }
    }
  }
}