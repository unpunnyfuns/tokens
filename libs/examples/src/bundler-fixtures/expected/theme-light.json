{
  "color": {
    "primary": {
      "$value": {
        "colorSpace": "srgb",
        "components": [0, 0.498, 1],
        "alpha": 1
      },
      "$type": "color"
    },
    "secondary": {
      "$value": {
        "colorSpace": "srgb",
        "components": [0.498, 0, 1],
        "alpha": 1
      },
      "$type": "color"
    },
    "neutral": {
      "100": {
        "$value": {
          "colorSpace": "srgb",
          "components": [0.961, 0.961, 0.961],
          "alpha": 1
        },
        "$type": "color"
      },
      "900": {
        "$value": {
          "colorSpace": "srgb",
          "components": [0.129, 0.129, 0.129],
          "alpha": 1
        },
        "$type": "color"
      }
    },
    "background": {
      "primary": {
        "$value": {
          "alpha": 1,
          "colorSpace": "srgb",
          "components": [0.961, 0.961, 0.961]
        },
        "$type": "color"
      },
      "secondary": {
        "$value": {
          "alpha": 1,
          "colorSpace": "srgb",
          "components": [0.129, 0.129, 0.129]
        },
        "$type": "color"
      }
    },
    "text": {
      "link": {
        "$value": {
          "alpha": 1,
          "colorSpace": "srgb",
          "components": [0, 0.498, 1]
        },
        "$type": "color"
      },
      "primary": {
        "$value": {
          "alpha": 1,
          "colorSpace": "srgb",
          "components": [0.129, 0.129, 0.129]
        },
        "$type": "color"
      },
      "secondary": {
        "$value": {
          "alpha": 1,
          "colorSpace": "srgb",
          "components": [0.961, 0.961, 0.961]
        },
        "$type": "color"
      }
    },
    "button": {
      "primary": {
        "background": {
          "$value": {
            "alpha": 1,
            "colorSpace": "srgb",
            "components": [0, 0.498, 1]
          },
          "$type": "color"
        },
        "text": {
          "$value": {
            "alpha": 1,
            "colorSpace": "srgb",
            "components": [0.961, 0.961, 0.961]
          },
          "$type": "color"
        }
      },
      "secondary": {
        "background": {
          "$value": {
            "alpha": 1,
            "colorSpace": "srgb",
            "components": [0.498, 0, 1]
          },
          "$type": "color"
        },
        "text": {
          "$value": {
            "alpha": 1,
            "colorSpace": "srgb",
            "components": [0.961, 0.961, 0.961]
          },
          "$type": "color"
        }
      }
    }
  }
}
